testsuite/shared/src/main/java/org/jboss/as/test/deployment/DeploymentInfoUtils.java

```diff
@@ -0,0 +1,782 @@
+    public enum DeploymentState {
+        // Statuses of Domain
+        ENABLED("enabled"), // Represent installed in selected server group and enabled application deployment
+        ADDED("added"), // Represent installed in selected server group but disabled application deployment
+        NOT_ADDED("not added"), // Represents application deployment of other server group that selected server group
+
+        // Statuses of Standalone
+        OK("OK"), // Represent enabled application deployment
+        STOPPED("STOPPED"), // Represent disabled application deployment
+
+        // Error status
+        UNKNOWN("!--unknown--!"); // Default value if isn't found
+
+        private String title;
+
+        DeploymentState(String title) {
+            this.title = title;
+        }
+
+        public String getTitle() {
+            return title;
+        }
+
+        /**
+         * Return information about enabled or disabled state from deployment info command statuses
+         *
+         * @param status Status of application deployment
+         * @return true application deployment id enabled, false is disabled
+         */
+        public static boolean mapBooleanByDeploymentStatus(@NotNull DeploymentState status) {
+            switch (status) {
+                case OK:
+                case ENABLED:
+                    return true;
+                case ADDED:
+                case STOPPED:
+                    return false;
+                case NOT_ADDED:
+                case UNKNOWN:
+                default:
+                    throw new IllegalArgumentException("Unsupported state " + status + "!");
+            }
+        }
+    }
+
+    // #### BEGIN Internal containers classes
+
+    /**
+     * The result of the deployment list/info invocation.
+     * Contains the output of the command that can be used to check the state of deployments.
+     */
+    public static class DeploymentInfoResult {
+        // Called command list or info about applications deployments for checking
+        private final String command;
+        // Selected server group in list/info command
+        private final String serverGroup;
+        // Original output of list/info command
+        private final String originalOutput;
+        /* Holding output of called command for multiple checking output without recall command
+           Output is parsed for processing*/
+        private final List<String> rows;
+        // Represent request of management operations to verify check
+        private String request;
+        // Represent response of management operations to verify check
+        private String response;
+
+        private DeploymentInfoResult(String command, String serverGroup, String output) {
+            this.command = command;
+            this.serverGroup = serverGroup;
+            originalOutput = output;
+            if (originalOutput != null && !originalOutput.isEmpty()) {
+                log.info("Read output:\n" + output);
```

Comment:
This is trace at maxium. not info.
Any info log ends up in end log files, for lots of jobs each day result in needles IO operations on CI.

Reply:
Done